newton/tests/test_import_usd.py

```
@@ -2424,6 +2424,188 @@ def PhysicsRevoluteJoint "Joint2"
+        prismatic_joint_idx = None
+        for i, jt in enumerate(model.joint_type.numpy()):
+            if jt == JointType.PRISMATIC:
+                prismatic_joint_idx = i
+                break
+        self.assertIsNotNone(prismatic_joint_idx, "No prismatic joint found")
+        self.assertAlmostEqual(dof_ref[qd_start[prismatic_joint_idx]], 0.5, places=4)
+
+    @unittest.skipUnless(USD_AVAILABLE, "Requires usd-core")
+    def test_springref_attribute_parsing(self):
+        """Test that 'mjc:springref' attribute is parsed for revolute and prismatic joints."""
+        from pxr import Usd  # noqa: PLC0415
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)